tools/SchemaManager/Commands/ApplyCommand.cs

```diff
@@ -150,39 +150,39 @@ await Policy.Handle<SchemaManagerException>()
             }
         }
 
-        private static void UpgradeSchema(string connectionString, int version, string script)
+        private static async Task UpgradeSchemaAsync(string connectionString, int version, string script, CancellationToken cancellationToken)
         {
             // check if the record for given version exists in failed status
-            SchemaDataStore.DeleteSchemaVersion(connectionString, version, SchemaDataStore.Failed);
+            await SchemaDataStore.DeleteSchemaVersionAsync(connectionString, version, SchemaDataStore.Failed, cancellationToken);
 
-            SchemaDataStore.ExecuteScriptAndCompleteSchemaVersion(connectionString, script, version);
+            await SchemaDataStore.ExecuteScriptAndCompleteSchemaVersionAsync(connectionString, script, version, cancellationToken);
 
             Console.WriteLine(string.Format(Resources.SchemaMigrationSuccessMessage, version));
         }
 
-        private static async Task<string> GetScript(ISchemaClient schemaClient, int version, string scriptUri, string diffUri = null)
+        private static async Task<string> GetScriptAsync(ISchemaClient schemaClient, int version, string scriptUri, CancellationToken cancellationToken, string diffUri = null)
         {
             if (version == 1)
             {
-                return await schemaClient.GetScript(new Uri(scriptUri, UriKind.Relative));
+                return await schemaClient.GetScriptAsync(new Uri(scriptUri, UriKind.Relative), cancellationToken);
             }
 
-            return await schemaClient.GetDiffScript(new Uri(diffUri, UriKind.Relative));
+            return await schemaClient.GetDiffScriptAsync(new Uri(diffUri, UriKind.Relative), cancellationToken);
         }
 
-        private static async Task ValidateVersionCompatibility(ISchemaClient schemaClient, int maxAvailableVersion)
+        private static async Task ValidateVersionCompatibility(ISchemaClient schemaClient, int maxAvailableVersion, CancellationToken cancellationToken)
         {
-            CompatibleVersion compatibleVersion = await schemaClient.GetCompatibility();
+            CompatibleVersion compatibleVersion = await schemaClient.GetCompatibilityAsync(cancellationToken);
 
             if (maxAvailableVersion > compatibleVersion.Max)
             {
                 throw new SchemaManagerException(string.Format(Resources.VersionIncompatibilityMessage, maxAvailableVersion));
             }
         }
 
-        private static async Task ValidateInstancesVersion(ISchemaClient schemaClient, int version)
+        private static async Task ValidateInstancesVersion(ISchemaClient schemaClient, int version, CancellationToken cancellationToken)
```

Comment:
Should this also have "async" in the name to be consistent?

Reply:
Thanks, fixed.